lnd_methods/offchain/probe_for_route.d.ts

```diff
@@ -1,70 +1,78 @@
-  /** Incoming Peer Public Key Hex String */
-  incoming_peer?: string;
-  /** Adjust Probe For Past Routing Failures Bool */
-  is_ignoring_past_failures?: boolean;
-  /** Maximum Fee Tokens Number */
-  max_fee?: number;
-  /** Maximum Fee Millitokens to Pay String */
-  max_fee_mtokens?: string;
-  /** Maximum Height of Payment Timeout Number */
-  max_timeout_height?: number;
-  messages?: {
-    /** Message To Final Destination Type Number String */
-    type: string;
-    /** Message To Final Destination Raw Value Hex Encoded String */
-    value: string;
-  }[];
-  /** Millitokens to Pay String */
-  mtokens?: string;
-  /** Outgoing Channel Id String */
-  outgoing_channel?: string;
-  /** Time to Spend On A Path Milliseconds Number */
-  path_timeout_ms?: number;
-  /** Payment Identifier Hex String */
-  payment?: string;
-  /** Probe Timeout Milliseconds Number */
-  probe_timeout_ms?: number;
-  routes?: {
-    /** Base Routing Fee In Millitokens Number */
-    base_fee_mtokens?: number;
-    /** Channel Capacity Tokens Number */
-    channel_capacity?: number;
-    /** Standard Format Channel Id String */
-    channel?: string;
-    /** CLTV Blocks Delta Number */
+export type ProbeForRouteArgs = AuthenticatedLightningArgs<
+  {
+    /** Final CLTV Delta Number */
     cltv_delta?: number;
-    /** Fee Rate In Millitokens Per Million Number */
-    fee_rate?: number;
-    /** Forward Edge Public Key Hex String */
-    public_key: string;
-  }[][];
-  /** Tokens Number */
-  tokens: number;
-  /** Total Millitokens Across Paths String */
-  total_mtokens?: string;
-}>;
+    /** Destination Public Key Hex String */
+    destination: string;
+    features?: {
+      /** Feature Bit Number */
+      bit: number;
+    }[];
+    ignore?: {
+      /** Channel Id String */
+      channel?: string;
+      /** Public Key Hex String */
+      from_public_key: string;
+      /** To Public Key Hex String */
+      to_public_key?: string;
+    }[];
+    /** Incoming Peer Public Key Hex String */
+    incoming_peer?: string;
+    /** Adjust Probe For Past Routing Failures Bool */
+    is_ignoring_past_failures?: boolean;
+    /** Maximum Fee Tokens Number */
+    max_fee?: number;
+    /** Maximum Fee Millitokens to Pay String */
+    max_fee_mtokens?: string;
+    /** Maximum Height of Payment Timeout Number */
+    max_timeout_height?: number;
+    messages?: {
+      /** Message To Final Destination Type Number String */
+      type: string;
+      /** Message To Final Destination Raw Value Hex Encoded String */
+      value: string;
+    }[];
+    /** Outgoing Channel Id String */
+    outgoing_channel?: string;
+    /** Time to Spend On A Path Milliseconds Number */
+    path_timeout_ms?: number;
+    /** Payment Identifier Hex String */
+    payment?: string;
+    /** Probe Timeout Milliseconds Number */
+    probe_timeout_ms?: number;
+    routes?: {
+      /** Base Routing Fee In Millitokens Number */
+      base_fee_mtokens?: number;
```

Comment:
shouldn't this be a string because decodePaymentRequest returns a string?

Reply:
fixed in [`005fa6f` (#119)](https://github.com/alexbosworth/lightning/pull/119/commits/005fa6f9d4c9cd72a9db5a4219934f5be0860295)